src/components/Form.js

```diff
@@ -36,16 +36,30 @@ const Form = () => {
 
   return (
     <form className="addform" onSubmit={submitHandler}>
-      <input name="title" type="text" placeholder="Book title" value={inputText.title} onChange={changeHandler} />
-      <input name="author" type="text" placeholder="Author" value={inputText.author} onChange={changeHandler} />
-      <select name="category" onChange={changeHandler} value={inputText.category}>
-        <option>Action</option>
-        <option>Fiction</option>
-        <option>Thriller</option>
-        <option>Inspiration</option>
+      <input
+        className="input-title"
+        name="title"
+        type="text"
+        placeholder="Book title"
+        value={inputText.title}
+        onChange={changeHandler}
+      />
+      <select
+        className="category-input"
+        name="category"
+        onChange={changeHandler}
+        value={inputText.category}
+      >
```

Comment:
- [ ] Kindly change the `background-color` of the category select field

Reply:
@moise10r You didn't specify what color I should change the category to

Reply:
@Yerimah use the same background-color you used for input field

Reply:
Resolved